app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Elf(Player, ABC):

```

Comment:
u don't need to inherit this class from ABC cause u have already inherit Player class from ABC

Reply:
is it normal that Pycharm points out an error
![Screenshot_241](https://user-images.githubusercontent.com/110467439/224299315-b3f56bd3-14b7-4287-873a-9bdbbc248058.png)